lib/cli/src/Cardano/CLI.hs

```diff
@@ -1296,6 +1299,21 @@ tlsOption = TlsConfiguration
         <> metavar "FILE"
         <> help "The RSA Server key which signed the x.509 server certificate."
 
+poolMetadataProxyOption
+    :: Parser URI
+poolMetadataProxyOption = option (eitherReader reader) $ mempty
+    <> long "pool-metadata-proxy"
```

Comment:
I would prefer to not use the word "proxy" where it could be confused with actual HTTP proxies.
We could just say `--smash-server`.
And what about making the CF smash server the default, with an option to disable the metadata aggregator?

Reply:
> I would prefer to not use the word "proxy" where it could be confused with actual HTTP proxies.

Fair point. Although I dislike the "--smash-server" naming for this because in principle, smash is one implementation of a metadata registry. There could be others by the community that aren't smash. Hence why I wanted to keep the name rather "generic". Perhaps `--pool-metadata-registry` ?

>  And what about making the CF smash server the default, with an option to disable the metadata aggregator?

There's no CF server yet as far as I know :grimacing: ... plus, with all the networks and candidates at the moment, it's sort of hard to know what a sensible default would be (other than, no default). Long-term, I do however agree, once the dust settle down a bit.

Reply:
I thought "smash" _was_ a generic name, or at least something that implements the smash API.
I have asked for clarification input-output-hk/smash#22.

Reply:
Anyway, I'll for "--smash-server" since it's more transparent for now. We can always change later.